packages/legend-application-studio/src/components/editor/edit-panel/uml-editor/TaggedValueEditor.tsx

```diff
@@ -45,12 +67,22 @@ interface TagOption {
 
 export const TaggedValueEditor = observer(
   (props: {
+    _whoKnows: unknown;
```

Comment:
First of all, let's avoid `unknown` unless we absolutely have to, the property one here is `AnnotatedElement`. That asides, instead of passing `whoKnows`, why don't we pass in the `rearrange` function only?

Reply:
Ah thanks switching to `AnnotatedElement`! On the passing just a function, when TaggedValueEditor is declared in multiple areas such as the ClassEditor and FunctionEditor, although I can pass in a custom function the rearrange function not only needs annotated element but requires `draggingProperty` and `HoveredProperty` as well with info not created until TaggedValueEditor